app/server.jsx

```diff
@@ -113,6 +113,11 @@ export default function render(req, res) {
               ${header.link.toString()}
             </head>
             <body>
+              <script>
+                if ('serviceWorker' in navigator) {
+                  navigator.serviceWorker.register('/sw.js')
```

Comment:
Can definitely be moved - possibly shouldn't be inlined at all and be part of app instead - this was just a dumping ground for now. Same as the catchall runtimeCaching.

Will make some more updates soon, I'll aim to get it into a mergeable state